scripts/openshiftci-presubmit-unittests.sh

```diff
@@ -11,7 +11,18 @@ export PATH=$PATH:$GOPATH/bin
 # otherwise /.cache is used, and it fails on permission denied
 export GOLANGCI_LINT_CACHE="/tmp/.cache"
 
-make goget-tools
+ARCH=`uname -i`
+if [ "$ARCH" = "x86_64" ]; then
+  export GOARCH=amd64
+else
+  export GOARCH=$ARCH
+fi
+
+wget https://github.com/golangci/golangci-lint/releases/download/v1.37.0/golangci-lint-1.37.0-linux-$GOARCH.tar.gz
+tar --no-same-owner -xzf golangci-lint-1.37.0-linux-amd64.tar.gz
+mv golangci-lint-1.37.0-linux-amd64/golangci-lint $(go env GOPATH)/bin/golangci-lint
+chmod +x $(go env GOPATH)/bin/golangci-lint
```

Comment:
Wouldn't it make more sense to change the `goget-tools` target instead to install `golangci-lint` via `wget` + `tar`? This way, we would have a single place to look at if we have to change the version of `golangci-lint` in the future. And we would be sure to have the exact version locally and in all our test platforms.

Reply:
Also, I think it would be nice to extract the version of `golangci-lint` as a local variable, so it can be easily updated in the future.